cypress/cypress-nginx.conf

```diff
@@ -0,0 +1,131 @@
+# NGINX Conf file used by Cypress-Nginx Github actions
+# so we can run cypress tests against the local nginx server
+# with a ssl cert set on the domain
+
+#user  nobody;
+worker_processes  1;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    include       mime.types;
+    default_type  application/octet-stream;
+
+    sendfile        on;
+
+    keepalive_timeout  65;
```

Comment:
Strange number!

Reply:
I've added a comment here. It _is_ a strange number.